stistools/response.py

```diff
@@ -0,0 +1,105 @@
+import math
+import numpy as np
+from astropy.io import fits
+
+import fit1d
```

Comment:
This import won't work with the installed package.

Maybe use something like:
`from .fit1d import fit1d`